package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "sprint-challenge-rdbms",
+  "version": "1.0.0",
+  "description": "This challenge allows you to practice the concepts and techniques learned over the past Sprint and apply them in a concrete project. This Sprint explored Adding Data Persistence to Web API's and you were taught the following modules: Introduction to Relational Databases and SQL, Inserting and Modifying Data, Querying Data; Migrations and Seeds and Introduction to Data Modeling. In your challenge for this Sprint, you will demonstrate proficiency by creating an API that pesist data to SQLite3.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Typically you'll see a script added in here for nodemon (otherwise there's not much of a reason to have it added as a dev dependency and all). For example:
```javascript
"scripts": {
  "start": "nodemon index.js"
},
```
or, using the `main` you have which is already index.js:
```javascript
"scripts": {
  "start": "nodemon"
},
```